src/randoop/reflection/OperationExtractor.java

```diff
@@ -0,0 +1,134 @@
+package randoop.reflection;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
+
+import randoop.operation.ConstructorCall;
+import randoop.operation.EnumConstant;
+import randoop.operation.FieldGetter;
+import randoop.operation.FieldSetter;
+import randoop.operation.InstanceField;
+import randoop.operation.MethodCall;
+import randoop.operation.Operation;
+import randoop.operation.StaticField;
+import randoop.operation.StaticFinalField;
+
+/**
+ * OperationExtractor is a {@link ClassVisitor} that creates a collection of {@link Operation}
+ * objects through its visit methods as called by {@link ReflectionManager#apply(Class)}.
+ * 
+ * @see ReflectionManager
+ * @see ClassVisitor
+ *
+ */
+public class OperationExtractor implements ClassVisitor {
+
+  private Set<Operation> operations;
+ 
+  /**
+   * OperationExtractor() creates a visitor object that collects Operation objects corresponding
+   * to class members visited by {@link ReflectionManager}. Stores {@link Operation} objects in
+   * ordered set to ensure strict order once flattened to list --- needed to guarantee determinism
```

Comment:
missing "an" before "ordered".

"ordered set" is an oxymoron, and I think you mean "ordered collection".